core/src/main/java/edu/wpi/grip/core/Step.java

```diff
@@ -82,7 +82,7 @@ public OperationDescription getOperationDescription() {
   }
 
   /**
-   * @return a UUID to distinguish this class
+   * @return A UUID to distinguish this class.
```

Comment:
First words in Javadocs for `@` lines (`@param`, `@return`, etc.) should be lowercase